Gemfile

```diff
@@ -3,9 +3,9 @@ source 'https://rubygems.org'
 ruby File.read('.ruby-version').strip
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.4'
+gem 'rails', '5.0.4'
```

Comment:
Notice that some of our other apps are using `5.1` - e.g. https://github.com/alphagov/publishing-api/blob/master/Gemfile#L4
and I notice `5.1.2` is out (http://weblog.rubyonrails.org/2017/6/26/Rails-5-1-2-has-been-released/)
Is this a conscious decision to use `5.0.4`?